kernel/Kconfig

```diff
@@ -192,6 +192,11 @@ config THREAD_STACK_INFO
 	  This option allows each thread to store the thread stack info into
 	  the k_thread data structure.
 
+config THREAD_ABORT_HOOK
+	bool "Custom hook when aborting thread"
+	help
+	  Used by portability layers to modify locally managed status mask.
```

Comment:
Please correct me if I am wrong ....

1. Doesn't the prompt suggest that this is user-facing?
2. If it is intended to be user-facing, then wouldn't we need to document the hook API and its signature?

As a potential alternative ... what if we used a weak function defined in sched.c and had a strong version of it defined in the CMSIS v1?

Reply:
> 1. Doesn't the prompt suggest that this is user-facing?

yes, removed